GETTING_STARTED.md

```diff
@@ -52,14 +48,12 @@ This will create the "ofn" user as superuser and allowing it to create databases
 
 Once done, run `script/setup`. If the script succeeds you're ready to start developing. If not, take a look at the output as it should be informative enough to help you troubleshoot.
 
-If you run into any other issues getting your local environment up and running please consult [the wiki][wiki].
-
-If still you get stuck do not hesitate to open an issue reporting the full output of the script.
-
 Now, your dreams of spinning up a development server can be realised:
 
     bundle exec rails server
 
+Go to [http://localhost:3000](http://localhost:3000) to play around!
+
 To login as the default user, use:
 
     email: ofn@example.com
```

Comment:
Not sure if it's just me but I found that `spree@example.com|spree123` worked instead of this credential.

Reply:
It was recently changed, you must have had a version of master with the previous version.

Reply:
If your environment was set up before the change, it won't be overwritten

Reply:
ah, yes, until the first db:reset :+1:

Reply:
Ace